src/components/molecules/FormFields/advancedSettings.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement, useState, FormEvent, ChangeEvent } from 'react'
+import Input from '../../atoms/Input'
+import Button from '../../atoms/Button'
+import { FormContent, FormFieldProps } from '../../../@types/Form'
+import { Field } from 'formik'
+import appConfig from '../../../../app.config'
+import styles from './advancedSettings.module.css'
```

Comment:
consistency, our React component names and their file names are in `PascalCase` so this component should be `AdvancedSettings.tsx` & `AdvancedSettings.module.css`

Reply:
This has been done